public/GarrysMod/GModDataPack.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <vector>
+#include <string>
+
+class LuaFindResult;
+
+abstract_class GModDataPack
+{
+public:
+	virtual void* GetFromDatatable( std::string const& ) = 0;
+	virtual void* GetHashFromDatatable( std::string const& ) = 0;
+	virtual void* GetHashFromString( char const*, unsigned long ) = 0;
+	virtual void FindInDatatable( std::string const&, std::vector<LuaFindResult>&, bool ) = 0;
+	virtual void* FindFileInDatatable( std::string const&, bool, bool ) = 0;
+	virtual bool IsSingleplayer() = 0;
+	virtual void v000000000000000000000000000oo0000000000000000fff000000000000000000000000000000000o0o() = 0;
```

Comment:
Can you get a better name for this one? If you don't know its name, I'd rather it be named something like `UnknownMethod`.

Reply:
It's how it's named in Gmod but I can change it
![image](https://github.com/danielga/sourcesdk-minimal/assets/64648134/1e28105a-b459-4516-b821-12f305081d89)

Reply:
I changed it to UnknownMethod but also left a comemnt with the original name